src/Event/Http/Psr15Handler.php

```diff
@@ -18,6 +18,8 @@ public function __construct(RequestHandlerInterface $psr15Handler)
     public function handleRequest(HttpRequestEvent $event, Context $context): HttpResponse
     {
         $request = Psr7RequestFactory::fromEvent($event);
+        $request = $request->withAttribute('lambdaEvent', $event);
```

Comment:
Does it make sense to expose the event? If one needs that, they can extend `HttpHandler` directly now instead of the PSR-7 handler. I don't think it brings added value.

Reply:
Well, there are a lot of batteries included in the existing implementation.
`\Bref\Event\Http\Psr7RequestFactory::fromEvent()` is marked as internal, so is
`\Bref\Event\Http\HttpResponse::fromPsr7Response();`, so if I want to replicate this behaviour I will need to write a lot of redundant code. I think it definitely makes sense to give the application direct access to the request context.

Reply:
Right, I did not mean about the request context: adding this one makes sense, because else there is no way to retrieve some stuff like Cognito authentication.

But I am questioning included the `lambdaEvent` as well: what is the use case for this?

Reply:
I'm not sure I understand what you mean - the Context contains the `\Bref\Context\Context`, right? How will that pass on the Cognito authentication? That looks like it's specified in the event array.

Reply:
Oh I'm sorry, maybe I'm mistaken!

OK here is what I assumed: both the `$context` variable and `$event['requestContext']` are the same thing. Are these different?

Reply:
Correct. `Context` is the `\Bref\Context\Context` object, which contains request metadata fields request ID, the function ARN, trace ID and the timeout deadline.

`$event` is the data passed to the function body. Here are some concrete of fields I have used today while working with PSR-7 handlers
- `$event["pathParameters"]` - I could parse the query string, but this is already parsed by API Gateway
- `$event["requestContext"]["authorizer"]["principalId"]` - the Authorizer user ID

Reply:
OK thank you that makes sense then!